api/_utils.ts

```diff
@@ -2061,13 +2078,13 @@ export async function fetchStakingPool(
 export const ConvertDecimals = (fromDecimals: number, toDecimals: number) => {
   // amount: string, BN, number - integer amount in fromDecimals smallest unit that want to convert toDecimals
   // returns: string with toDecimals in smallest unit
-  return (amount: BigNumber): string => {
+  return (amount: BigNumber) => {
```

Comment:
```suggestion
  return (amount: BigNumber): BigNumber => {
```

Also specify in comment above